security/pass-otp/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        tadfisher pass-otp 1.1.1 v
+revision            1
+maintainers         loicp.eu:loic-github
```

Comment:
Please also add your github handle with the following format:
```
{loicp.eu:loic-github @lpefferkorn}
```

Reply:
Sure!